mkdocs.yml

```diff
@@ -17,9 +17,12 @@ nav:
       - Community : Projects/community-proj.md
       - Muscle / Neuron /  Channel : Projects/muscle-neuron-integration.md
   - Community Overview : community.md
-  - Community: 
+  - Community:
       - Github : Community/github.md
       - Repos : Community/repositories.md
       - Membership : Community/membership.md
-  - Resources : Resources/resources.md
+  - Resources:
+      - Resources: Resources/resources.md
+      - Running NeuroML connectome: Resources/running-nc.md
+      - Live video streaming: Live-Video-Protocol.md
```

Comment:
Honestly, now that I'm looking at this file, it looks like it's just part of Bradly's workflow -- it doesn't belong in these docs anyway.

Reply:
@mwatts15 which file? Live-Video-Protocol.md? Resources/running-nc.md? both?

Reply:
removed Live-Video-Protocol.md as per slack message